lib/ojs.mli

```diff
@@ -83,26 +101,17 @@ external apply: t -> t array -> t = "caml_js_fun_call"
 external new_obj: t -> t array -> t = "caml_js_new"
 (** Call a constructor *)
 
-val call_arr: t -> string -> t -> t
```

Comment:
Is the removal of those function related to the PR?

Reply:
I could keep them there (even though the ppx is not using them anymore) (but to keep I would need to replace the use of "get" by "get_prop  + string_to_js")